portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Returns URL that allows a user to upload a file to Blobstore.
+ */
+@WebServlet("/blobstore-upload-url")
+public class BlobstoreUploadUrlServlet extends HttpServlet {
+    BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        String uploadUrl = blobstoreService.createUploadUrl("/comments");
+        response.setContentType("text/html");
```

Comment:
This still works, but it would be good to align with the actual content that is sent.
For example, you can also use text/plain.
For more info, please refer to the available MIME types.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types